docs/docs/30-administration/40-encryption.md

```diff
@@ -1,14 +1,23 @@
 # Secrets encryption
 
-By default, Woodpecker does not encrypt secrets in its database. You can enable encryption 
+:::danger
```

Comment:
That this information was missing till now, and that it was just disabled in the code while users were still able to configure it without any notice e.g. by adding at least a warning to the log that encryption was configured but doesn't work is IMO pretty, pretty bad.

Reply:
It is not the best that way, but that is the reality what happens in larger projects from time to time with a lot of people working on different parts I guess 🤷🏾‍♂️.